security/EvidenceSecurity.hs

```diff
@@ -104,7 +104,14 @@ theParser = some line <* eof
     notColonOrNewline = noneOf [':','\n']
 
 
-data Category = Authorization | Privacy | Semantics | Performance | InputValidation | Authentication
+data Category
+  = Authentication
+  | Authorization
+  | Availability
+  | Confidentiality
+  | Integrity
+  -- | NonRepudiation
```

Comment:
Should these be removed instead of commented?

Reply:
Fixed.